### ISSUE TITLE
Copy an array to target root

### ISSUE THREAD

**Issue by SaviChinnadurai** [WorkMaze/JUST.net#161]
Hi,

I have a requirement to do transformation as below but not sure if it is possible. Any suggestions?

Input Data:
{
	"Food": {
		"Desserts": {
			"item": [
				{
					"name": "carrot cake",
					"price": 5
				},
				{
					"name": "ice cream",
					"price": 10
				}
			]
		}
	}
}         

Required Output Data:
[
	{
		"name": "carrot cake",
		"price": 5
	},
	{
		"name": "ice cream",
		"price": 10
	}
]

I am able to get the array mapped into below structure though.

{
	"items": [
		{
			"name": "carrot cake",
			"price": 5
		},
		{
			"name": "ice cream",
			"price": 10
		}
	]
}

Thanks.

**Comment by Courela** [comment]
I don't think it is possible, a root property is necessary as you mentioned (items).
